tools/release/README.md

```diff
@@ -52,6 +52,8 @@ A Quality Assurance Engineer from each product group needs to [confirm that thei
 
 **3. Merge changelog and version bump**
 
+Manally increment the [Helm chart]([url](https://github.com/fleetdm/fleet/blob/main/charts/fleet/Chart.yaml)) minor version by 1 and push to your changelog PRs.
```

Comment:
Sorry I was unclear. This only goes up minor versions if the values in the chart templates or values.yaml changes. If we are only updating the appversion (typical) then even for minor release the patch should increment